packages/dapp/src/helpers/queryTransformations.ts

```diff
@@ -107,7 +114,7 @@ export function transformGraphQLDataIntoListing(listing: any, listingAddress: st
 export function transformGraphQLDataIntoChallenge(queryChallengeData: any): ChallengeData | undefined {
   if (queryChallengeData) {
     return {
-      statement: queryChallengeData.statement,
+      statement: "",
```

Comment:
Is the statement not currently in the GraphQL response? It looks like it's specified in `LISTING_FRAGMENT`...

Reply:
going to be dealing with that in my next PR, it was coming down in a way that broke rendering

Reply:
👍